src/main/java/ReviewResponse.java

```diff
@@ -1,7 +1,17 @@
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+@ToString
+@Data
 public class ReviewResponse {
```

Comment:
Violation of Single Responsibility Principle (SRP) - This class has mixed responsibilities (data representation and behavior). It should be split into separate classes for better cohesion. (Severity: high)